colossus/colossus-htb.js

```diff
@@ -175,9 +175,11 @@ function ColossusHtb(configs) {
          * returned from gdpr.getConsent() are safe defaults and no attempt has been
          * made by the wrapper to contact a Consent Management Platform.
          */
-        queryObj.gdprStatus = ComplianceService.gdpr.getConsent();
-        queryObj.privacyEnabled = ComplianceService.isPrivacyEnabled();
-
+        if (ComplianceService) {
+            queryObj.gdprStatus = ComplianceService.gdpr.getConsent();
+            queryObj.privacyEnabled = ComplianceService.isPrivacyEnabled();
+            queryObj.ccpa = ComplianceService.usp && ComplianceService.usp.getConsent();
+        }  
```

Comment:
Hi,
Thanks for fixing the previous linting error.
However, there seem to be 2 new ones created in the process.

`182:10  error  Trailing spaces not allowed   no-trailing-spaces`
`183:9   error  Expected line before comment  lines-around-comment`

You can run the lint test as instructed in the link below.
https://kb.indexexchange.com/Adapters/ESLint.htm

Thanks!

Reply:
Done